pycbc/waveform/generator.py

```diff
@@ -283,6 +291,20 @@ def __init__(self, variable_args=(), **frozen_params):
             variable_args=variable_args, **frozen_params)
 
 
+class FDomainCBCModesGenerator(BaseCBCGenerator):
+    """Generates frequency-domain CBC waveform modes in the radiation frame.
+
+    Uses :py:func:`waveform_modes.get_fd_waveform_modes` as a generator
+    function to create frequency-domain CBC waveforms in the radiation frame;
+    i.e., with no detector response function applied. For more details, see
```

Comment:
Fix comment here to reflect source frame. No detector response function *or* source orientation w.r.t. the observer applied.

Reply:
Fixed. As I said above, source frame might be misleading too, since the masses are detector frame. I just dropped "radiation frame", and just said it generates mode-by-mode, without applying spherical harmonics.